e2e/test/scenarios/visualizations-tabular/drillthroughs/chart_drill.cy.spec.js

```diff
@@ -103,6 +104,12 @@ describe("scenarios > visualizations > drillthroughs > chart drill", () => {
 
       cy.createQuestion(questionDetails, { visitQuestion: true });
 
+      queryBuilderMain().within(() => {
+        cy.findByLabelText("Legend").findByText("Gadget").should("exist");
+        echartsContainer().findByText(/Count/).should("exist");
+      });
+      cy.wait(100); // wait to avoid grabbing the svg before the chart redraws
```

Comment:
While I don't love the hardcoded wait, this was the pattern established by other tests